src/python/WMComponent/DBS3Buffer/DBSBufferBlock.py

```diff
@@ -455,14 +456,14 @@ def FillFromDBSBuffer(self, blockInfo):
         self.startTime = blockInfo.get('creation_date')
         self.inBuff    = True
 
-        if 'status' in blockInfo.keys():
+        if 'status' in blockInfo.keys():  # did not modernize, lack of test
```

Comment:
```suggestion
        if 'status' in blockInfo:
```
can it be something like that?

Reply:
Yeah! you are correct. I saw that `.get()` function and imagined that `blockInfo` was some exotic class, but only now I realize that it is the safe way of accessing a the value given the key in a normal python dict. Thanks! (corrected in last push)